lifelines/fitters/aalen_additive_fitter.py

```diff
@@ -443,7 +443,7 @@ def plot_coefficients_bar_summary(self, **kwargs):
         from matplotlib import pyplot as plt
         ax2 = kwargs.get('ax2', None) or plt.figure().add_subplot(111)
         x_bar = self.hazards_.sum(axis=0)
-        x_bar.plot(ax=ax2, kind='barh')
+        x_bar.plot(ax=ax2, kind='bar', yerr=0.1)
```

Comment:
is that the right error though?

Reply:
Hmm - I'm not sure. what's the right error?

Reply:
Ahh I get it :)